tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": "@sindresorhus/tsconfig",
+	"compilerOptions": {
+		"esModuleInterop": true,
+		"noImplicitAny": true,
+		"declaration": true,
+		"allowSyntheticDefaultImports": true,
```

Comment:
From experience, this option causes lots of weird problems. Better to leave it out.